Unit-01/03-templating/app.py

```diff
@@ -0,0 +1,33 @@
+from flask import Flask, render_template, request, url_for
+
+app = Flask(__name__)
+
+
+# Part 1
+@app.route('/person/<name>/<int:age>')
+def say_name_and_age(name, age):
+	name = name.title()
+	return render_template('person.html', name=name, age=age)
+	
+# Part 2
+@app.route('/calculate')
+def calc():
+	return render_template('calc.html')
+
+@app.route('/math')
+def math():
+	calculation = request.args.get('calculation')
+	num1 = int(request.args.get('num1'))
+	num2 = int(request.args.get('num2'))
+
+	if calculation == 'add':
+		return "The sum is: " + str(num1 + num2)
+	elif calculation == 'subtract':
```

Comment:
This is good; However, another way to do this is using a dictionary, just like you can in JavaScript using objects.

```python
def add(a, b):
    return a + b
def subtract(a, b):
    return a - b
def multiply(a, b):
    return a * b
def divide(a / b):
    return a / b

operations = {
    "add": add,
    "subtract": subtract,
    "multiply": multiply,
    "divide": divide
}